charter.md

```diff
@@ -72,28 +78,159 @@ this works.
-a special election will serve out the remainder of the term for the person they
-are replacing, regardless of the length of that remainder.
+be held as soon as possible.
+
+Examples (non-exhaustive):
+
+* the next most preferred candidate is no longer interested in sitting on the
+  committee
+* the next most preferred candidate cannot sit on the committee, due to some
+  committee composition requirement i.e.,
+  [maximal representation][maximal-representation]
 
+[Eligible voters][voter-eligibility] from the most recent election will vote in
+the special election i.e., eligibility will not be redetermined at the time of
+the special election.
+
+A committee member elected in a special election will serve out the remainder
+of the term for the person they are replacing, regardless of the length of that
+remainder.
+
+[maximal-representation]: /elections.md#maximal-representation
 [voter-eligibility]: /elections.md#eligibility-for-voting
 
-## Quorum
+### Resignation
+
+If a committee member chooses not to continue in their role, for whatever
+self-elected reason, they must notify the committee in writing.
+
+### Removal
+
+#### No confidence
+
+A Steering Committee member may be removed by an affirmative vote of a
+**_three-quarters supermajority of the
+[fixed membership of the committee](#composition)_**.
+
+Example:
+
+* 7 (members) / 4 = 1.75
+* 1.75 * 3 = 5.25
+* Round up to the nearest whole number (6)
+* Six (6) affirmative votes would be required to remove a member through a vote
+  of no confidence
+
+The call for a vote of no confidence will happen in a public Steering Committee
+meeting and must be documented as a GitHub issue in the committee's
+[repository][steering-repo].
+
+The call for a vote of no confidence must be made by a current member of the
+committee and must be seconded by another current member.
+
+The committee member who calls for the vote will prepare a statement which
+provides context on the reason for the vote. This statement must be seconded by
+the committee member who seconded the vote.
+
+Once a vote of no confidence has been called, the committee will notify the
+community through the following channels:
+
+* the [community mailing list][dev-list]
+* the [Steering Committee public mailing list][steering-public-list]
+
+This notification will include:
+
+* a link to the aforementioned GitHub issue
+* the statement providing context on the reason for the vote
+
+There will be a period of two weeks for members of the community to reach
+out to Steering Committee members to provide feedback.
+
+Community members may provide feedback by the following methods:
 
-Steering committee members are generally expected to attend every meeting. We
+* commenting on the GitHub issue
+* replying to the notification email
+* sending an email to the
+  [Steering Committee private mailing list][steering-private-list]
+* sending a message to individual committee members
```

Comment:
What should happen within the SC if someone picks this route? Or should folks make no assumption about folks forwarding the message on internally etc.